view/frontend/web/js/view/checkout/validator-mixin.js

```diff
@@ -15,10 +15,10 @@ define(['jquery'], function($) {
         $.validator.addMethod(
             "validate-ssn-br",
             function(value, element) {
-                // The format is 000.000.000-00
-                return /^([-\.\s]?(\d{3})){3}[-\.\s]?(\d{2})$/.test(value);
+                // The format is 000.000.000-00 (CPF) and 00.000.000/0000-00 (CNPJ)
+                return /(^\d{3}\.\d{3}\.\d{3}\-\d{2}$)|(^\d{2}\.\d{3}\.\d{3}\/\d{4}\-\d{2}$)/.test(value);
```

Comment:
We can split the expression for readability: 
```suggestion
                // The CPF format is: 000.000.000-00
                const cpf = /^\d{3}\.\d{3}\.\d{3}\-\d{2}$/.test(value);
                // The CNPJ format is: 00.000.000/0000-00
                const cnpj = /^\d{2}\.\d{3}\.\d{3}\/\d{4}\-\d{2}$/.test(value);
                return cpf || cnpj;
```

Reply:
@michaelpaul thank you, added!